notesapi/v1/models.py

```diff
@@ -11,12 +12,16 @@ class Note(models.Model):
     course_id = models.CharField(max_length=255, db_index=True)
     usage_id = models.CharField(max_length=255, help_text="ID of XBlock where the text comes from")
     quote = models.TextField(default="")
+    parent = models.ForeignKey('Note', blank=True, null=True, help_text="Parent note, if this is a comment")
```

Comment:
No worries, my bad. I shouldn't have been commenting without that tag. I just had the tab up in my browser from earlier, and when I flipped over to it I thought I'd get a head start on tomorrow.